src/Traits/Activatable.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CodeIgniter\Shield\Traits;
+
+trait Activatable
+{
+    /**
+     * Returns true if the user has been activated
+     * and activation is required after registration.
+     */
+    public function isActivated(): bool
+    {
+        // If activation is not required, then we're always active.
+        return ! $this->shouldActivate() || $this->active;
+    }
+
+    /**
+     * Returns true if the user has not been activated.
+     */
+    public function isNotActivated(): bool
+    {
+        return ! $this->isActivated();
+    }
+
+    /**
+     * Activates the user.
+     */
+    public function activate(): bool
+    {
+        helper('auth');
```

Comment:
```suggestion
```
Loaded by default, no need to load.

Reply:
I forgot it was loaded via Composer now. Was thinking it was still in the filters. Thanks for reminder.